qe_metrics/commands/init_config/__init__.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""Module building the init-config cli command"""
+
+from pathlib import Path
+
+import click
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+from simple_logger.logger import get_logger
+
+
+@click.argument(
+    "filepath",
+    type=click.Path(exists=False),
+    default="./qe-metrics.config",
+)
+@click.command("init-config")
+def init_config(filepath: str) -> None:
```

Comment:
1. there should be one entry point
2. why is this needed? a user needs to provide the info anyhow; create an example file `qe-metrics.config.example` with the required structure. A user just needs to fill it in

Reply:
1. This or hasn't been refactored to do that, I did it in https://github.com/RedHatQE/qe-metrics/pull/4 . Thats why this one is in WIP

2. Fine by me